ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo.hs

```diff
@@ -35,10 +35,9 @@ enumCoreNodes (NumCoreNodes numNodes) =
 -------------------------------------------------------------------------------}
 
 -- | Data required to run the specified protocol.
-data ProtocolInfo m b = ProtocolInfo {
+data ProtocolInfo b = ProtocolInfo {
```

Comment:
This type is a prominent part of our interface, and its used in much downstream code/repos.

1) Removing a now-dead parameter is a nice simplification, but it'll be onerous to integrate downstream. Perhaps we should open that PR (once this PR gets far enough through the process).

2) This PR should explain the change in [`interfaceCHAN](https://github.com/input-output-hk/ouroboros-network/blob/master/ouroboros-consensus/docs/interface-CHANGELOG.md)

Reply:
That's my plan, still we need to use the new interface in `cardano-node` to reload block forging.

Reply:
Yeah. The only alternative I've imagined (and which I'm not excited about) would be to leave the `m` on `ProtocolInfo`, even unused. Just to avoid having to update the type signatures.

Especially if we're the ones making the downstream PRs, then the confusion/etc around that is probably not worth it. But if others are coping with the change, then it might save them some work. Just pointing out the option.

Reply:
That's an easy change, in `cardano-node` there were just [one place](https://github.com/input-output-hk/cardano-node/pull/4024/commits/829fbc6761e31985e1c54df2a78af53a311b5322) to update, it's probably not that difficult in `cardano-wallet` and `cardano-db-sync`.

A draft pr in the node: https://github.com/input-output-hk/cardano-node/pull/4024

Reply:
I had overlooked the fact that this whole workstream required a Node PR. And as a bonus: the necessary changes in Node were indeed simple, for this signature change.

I hope it's that simple in other downstream code!